documentation/docs/05-modules.md

```diff
@@ -57,7 +57,15 @@ Because of that, the stores are not free-floating objects: they must be accessed
 The stores themselves attach to the correct context at the point of subscription, which means you can import and use them directly in components without boilerplate. However, it still needs to be called synchronously on component or page initialisation when `$`-prefix isn't used. Use `getStores` to safely `.subscribe` asynchronously instead.
 
 - `navigating` is a [readable store](https://svelte.dev/tutorial/readable-stores). When navigating starts, its value is `{ from, to }`, where `from` and `to` both mirror the `page` store value. When navigating finishes, its value reverts to `null`.
-- `page` is a readable store whose value reflects the object passed to `load` functions — it contains `host`, `path`, `params` and `query`. See the [`page` section](#loading-input-page) above for more details.
+- `page` is a readable store whose value reflects the object passed to `load` functions — it contains `host`, `path`, `params` and `query`. See the [`page` section](#loading-input-page) above for more details. Updating the query params of a page can be done by means of navigation by creating a new instance of `$page.query` and using the [goto](#modules-$app-navigation). For example:
```

Comment:
this seems like an odd place to add this detail

Reply:
Could you please suggest a better a place to add this detail? i

Reply:
I'm not sure where this will fit either, it sounds more like a tip than documentation. Putting it in the FAQ may be too niche too. What if we word this differently, like for [page](https://kit.svelte.dev/docs#loading-input-page) we can say that "mutating page does not update the current URL".

Reply:
alright, will remove this then.